pkg/windows/windows.go

```diff
@@ -395,7 +396,17 @@ func (vm *windows) RebootAndReinitialize() error {
 	if _, err := vm.Run("Restart-Computer -Force", true); err != nil {
 		return fmt.Errorf("error rebooting the Windows VM: %w", err)
 	}
-	// Reinitialize the SSH connection after the VM reboot
+
+	// Wait for instance to be unreachable via SSH, implies reboot is underway
+	err := wait.PollUntilContextTimeout(context.TODO(), retry.WindowsAPIInterval, retry.ResourceChangeTimeout, true,
+		func(ctx context.Context) (bool, error) {
+			return !vm.interact.isReachable(), nil
```

Comment:
Alternatively can't you keep calling `vm.reinitalize()` until it returns an error?

Reply:
I wanted a different Poll interval and timeout, though I guess this can be achieved by parameterizing those

Reply:
Rather than having poll interval and timeout as input you could have `reason` as input and pick the times based on that.